.eslintrc.cjs

```diff
@@ -1,15 +0,0 @@
-module.exports = {
```

Comment:
Please, revert this change. eslint shouldn't change here

Reply:
Okay, I'll roll it back. Thanks! :)